<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите базу данных:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_db_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить базу данных</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбранная база данных:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="db_filename_le">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите текст для анализа:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_text_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать текст</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_field">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>341</width>
     <height>251</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_sent_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать предложение:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sent_field">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="sg_cb">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>590</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="make_col_btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>590</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назначить сочетание</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="col_table">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column>
    <property name="text">
     <string>Сочетание</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Семантическая группа</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сочетания</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_col_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="union_col_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>450</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Объединить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_words_col_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>490</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить слова</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_sg_col_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>530</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить с.г.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="make_con_btn">
   <property name="geometry">
    <rect>
     <x>562</x>
     <y>410</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать пару</string>
   </property>
  </widget>
  <widget class="QPushButton" name="automake_con_btn">
   <property name="geometry">
    <rect>
     <x>562</x>
     <y>530</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Автосоздание пар</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="con_table">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>150</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column>
    <property name="text">
     <string>Предикат</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Актант</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пары</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_con_btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new_sg_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить новую с.г.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
